sandbox/src/mocks/r4/getServiceRequest/responses/adviceRequestedAandG.json

```diff
@@ -0,0 +1,101 @@
+{
+  "resourceType": "ServiceRequest",
+  "id": "a.832db7fa-ebdd-44b6-ab3b-8329c2d43149",
+  "meta": {
+    "versionId": "5"
+  },
+  "subject": {
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9912003888"
+    }
+  },
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "https://fhir.nhs.uk/CodeSystem/message-category-servicerequest",
+          "code": "referral"
+        }
+      ]
+    }
+  ],
+  "code": {
+    "coding": [
+      {
+        "system": "https://fhir.nhs.uk/CodeSystem/ServiceRequest-Code",
+        "code": "advice-and-guidance"
+      }
+    ]
+  },
+  "priority": "routine",
+  "intent": "order",
+  "status": "active",
+  "extension": [
+    {
+      "url": "https://fhir.nhs.uk/StructureDefinition/Extension-ERS-Referral",
+      "extension": [
+        {
+          "url": "sourceSystem",
+          "valueCode": "ers"
+        }
+      ]
+    },
+    {
+      "url": "https://fhir.nhs.uk/R4/StructureDefinition/Extension-eRS-ReferralState-1",
+      "extension": [
+        {
+          "url": "state",
+          "valueCoding": {
+            "system": "https://fhir.nhs.uk/R4/CodeSystem/eRS-ReferralState-1",
+            "code": "ACTIVE", ( do we need to mirror the ers states ( like AWAITING_RESPONSE)
+            "display": "Active"
+          }
+        }
+      ]
+    }
+  ],
+  "identifier": [
+    {
+      "system": "https://fhir.nhs.uk/Id/UBRN",
+      "value": "111111111112"
+    }
+  ],
+  "contained": [
+    {
+      "resourceType": "PractitionerRole",
+      "id": "requester",
+      "practitioner": {
+        "identifier": {
+          "system": "http://fhir.nhs.net/Id/sds-user-id",
+          "value": "103264992985"
+        },
+        "display": "DR AA BHATIA"
+      },
+      "organization": {
+        "identifier": {
+          "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+          "value": "C81007"
+        },
+        "display": "VERNON STREET MEDICAL CTR"
+      }
+    }
+  ],
+  "requester": {
+    "reference": "#requester"
+  },
+  "supportingInfo": [
```

Comment:
Yeah I like the forward references here. Then you don't have to go searching for things related to this ServiceRequest.